### ISSUE TITLE
challenges with google assistant step5

### ISSUE THREAD

**Issue by berazzi** [eouia/MMM-AssistantMk2#88]
When I run node auth_and_test.js it opens up a a browser alright, however for some reason I get this:

<img width="819" alt="screenshot 2019-02-14 at 19 40 37" src="https://user-images.githubusercontent.com/46160289/52809456-92ede980-3090-11e9-9dc0-e1aef3c0d722.png">

Which is referring to a different proejct number than mine. Which is weird as I did get the credentials from my project. Any idea what could have gone wrong?

Does there need to be something matching/something linked between the project on the actions console and the cloud platform? Because the project id's doesnt match at all, however as it is my first try with either I have no idea if this is an issue.

**Comment by eouia** [comment]
Your credentials.json might come from another project (nevertheless different with your beliefs)

Try again. If needed, remove projects and make newly.

**Comment by berazzi** [comment]
After some mucking back and forth I think I managed to atleast partially resolve this. Because now I get a bit further than before :-) I get prompted with login like shown here:
<img width="544" alt="screenshot 2019-02-15 at 19 55 31" src="https://user-images.githubusercontent.com/46160289/52878008-f2fd9200-315b-11e9-83a8-60e2a8196ffa.png">

I guess this is correct from the installation instructions, however nothing happens when I click my user. It's like it starts to load, but then nothing. Suggestions?

**Comment by eouia** [comment]
Maybe you didn't `enable` the Google Assistant API. Follow the instruction carefully again.

**Comment by berazzi** [comment]
I actually had'nt, missed it in my frustration over it not working. I started over, first creating project on the actions console, then in the cloud platform console, then enabling assistant api, then registering a device, downloading credentials and went via vnc to the mmM-AssistantMk2 folder and did node auth_and_test.js.
Still doing the same thing :-/
Is it related to how I registered my device, does it have to be a specific device  or type or something?
